### PR TITLE
Remove fixed dependencies on react & react-native

### PR DESCRIPTION
**Pull request by EmilScherdin**
See #9

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,9 +21,7 @@
   ],
   "author": "nlt2390",
   "dependencies": {
-    "prop-types": "15.6.0",
-    "react": "16.0.0",
-    "react-native": "0.50.1"
+    "prop-types": "^15.0.0",
   },
   "peerDependencies": {
     "react": "*",
```
Comment (nlt2390) [inline package.json]: can u add prop-types here too ?

Comment (EmilScherdin) [reply package.json]: Yep, they say you should add it there in the read me: https://github.com/facebook/prop-types#how-to-depend-on-this-package

Comment (EmilScherdin) [reply package.json]: Sorry for the delayed respose

Comment (nlt2390) [reply package.json]: I mean, plz make a new commit adding it to peerDependencies and I will merge.

**Comment by EmilScherdin** [comment]
@nlt2390 Fixed.

**Pull request merged**